common/config/rush/pnpm-lock.yaml

```diff
@@ -179,7 +179,7 @@ importers:
     specifiers:
       '@azure-tools/typespec-autorest': '>=0.31.0 <1.0.0'
       '@azure-tools/typespec-azure-core': '>=0.31.0 <1.0.0'
-      '@azure-tools/typespec-client-generator-core': '>=0.31.0 <1.0.0'
+      '@azure-tools/typespec-client-generator-core': file:/Users/zhangqiaoqiao/work/code/typespec-azure/packages/typespec-client-generator-core
```

Comment:
This is just for temporary, will need to update it when tcgc PR get merged and released.